google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -578,4 +578,18 @@
     <className>com/google/cloud/spanner/connection/Connection</className>
     <method>com.google.api.core.ApiFuture closeAsync()</method>
   </difference>
+
+  <!-- Clirr seems to fail for these methods when upgrading to Java 8 -->
+  <!-- Adding these to unblock the CI -->
+  <!-- FIXME: Remove the methods below once clirr is fixed -->
+  <difference>
```

Comment:
Clirr does not seem to support Java 8 (https://github.com/mojohaus/clirr-maven-plugin/issues/3). The methods just added to the `clirr-ignored-differences.xml` are not real breaking changes, they were available in the client since July 1st, 2020 (https://github.com/googleapis/java-spanner/commit/462839b625e58e235581b8ba10b398e1d222eaaf)